pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -5558,6 +5558,42 @@ Bottom Left.`,
 			Volatility: tree.VolatilityImmutable,
 		},
 	),
+	"st_makeenvelope": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"xmin", types.Float},
+				{"ymin", types.Float},
+				{"xmax", types.Float},
+				{"ymax", types.Float},
+				{"srid", types.Int},
+			},
+			ReturnType: tree.FixedReturnType(types.Geometry),
+			Fn: func(_ *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				return makeGeometryFromBounds(args)
+			},
+			Info: infoBuilder{
+				info: "Returns a box2d from bounds.",
```

Comment:
nit: this comment is incorrect.
"Creates a rectangular Polygon from the minimum and maximum values for X and Y with the given SRID."